python/amici/petab_import.py

```diff
@@ -227,6 +226,10 @@ def import_model(sbml_model: Union[str, 'libsbml.Model'],
     if verbose:
         logger.log(logging.INFO, f"{Fore.GREEN}Importing model ...")
 
+    # Create a copy, because it will be modified by SbmlImporter
+    sbml_doc = sbml_model.getSBMLDocument().clone()
```

Comment:
could be done in one line to stress that it is the sbml_model that you are interested in, not the sbml_doc.

Reply:
deep copy is not implemented?

Reply:
Didn't find deepcopy documented and didn't test.
We do need those two lines, otherwise the Model comes without a document which causes trouble later on.